bridge.py

```diff
@@ -106,13 +105,12 @@ def for_each_frame(self):
         elif pos.y < 0.0:
             if not self.train_off_screen:
                 self.sounds['death'].play()
-                print "TRAIN FELL OFF!", pos.x
+                print("TRAIN FELL OFF!", pos.x)
                 self.train_off_screen = True
 
-    def create_train(self, worldpoint=(1600, 490),
-                     train=(100, 50),
-                     wheelrad=20,
-                     cars=3,
+    def create_train(self, worldpoint=(int(1600*0.625), int(490*0.733)),
+                     train=(int(100*0.625), int(50*0.733)),
+                     wheelrad=int(20*0.733), cars=3,
```

Comment:
These constants could be calculated dynamically by comparing the current main window geometry against the original design geometry.

Reply:
Yes they are constants for adjusting the resolution, will define a global constant at the TOP.

Reply:
Done in 8694d00. Also I have edited my opening comment accordingly. Moreover, I feel the coordinates used by the activity were not for smaller displays like the one I tested on. Everything is out of focus for me, however, looking at @srevinsaju's screenshot, this might not be the case for a 1920x1080 display. I have used a scaling factor that has fixed this issue for me. Kindly test for different dimensions.

Reply:
Tested, fixed. Please mark as resolved